src/utils.ts

```diff
@@ -2038,11 +2128,22 @@ export class newTooltip {
       this.squareOrigin.x += upRightDiff.x - plotSize.x;
     }
     if (upRightDiff.y < 0) {
-      this.squareOrigin.y += upRightDiff.y;
-      this.origin.y += upRightDiff.y;
+      if (!this.isFlipper) {
```

Comment:
You could probably simplify if statements here

Reply:
changed order to avoid negation and put some on one line